articles/clients/index.md

```diff
@@ -4,186 +4,167 @@ description: Explains the basics of creating and using Auth0 Clients.
 
-1. Relate to your applications;
-2. Impact auditing, authorization, and so on.
+## How to configure a Client in Auth0
 
-## Client Settings
+Navigate to the [dashboard](${manage_url}) and click on the [Clients](${manage_url}/#/clients) menu option on the left. By default, you should have one client named *Default App*. You can either configure this one or create a new one by clicking the **+ Create Client** button.
 
-![](/media/articles/applications/applications-callback-settings.png)
+The *Create Client* windows pops open. Set a descriptive name for your client and select the client type.
 
-When you configure your Auth0 client, you will be asked for or provided (as indicated by a grayed out text box) the following pieces of information:
+![Create Client window](/media/articles/applications/create-client-popup.png)
 
-* **Name**: the name of your client (which you'll see in the portal, emails, logs, and so on);
-* **Domain**: the domain name of your client;
-* **Client ID**: the unique identifier for your client (this is the ID you'll use with when configuring authentication with Auth0). By default, the  is hidden, so check the **Reveal Client Secret** box to see this value;
-* **Client Secret**: the key used to sign and validate tokens for authentication flows and to gain access to select Auth0 API endpoints;
-* **Client Type**: the type of client you are implementing (depending on which you choose, the settings available differ);
-* **Token Endpoint Authentication Method**: the requested authentication method for the token endpoint: `None` (public client without a client secret), `Post` (client uses HTTP POST parameters), or `Basic` (client uses HTTP Basic parameters);
-* **Allowed Callback URLs**: the URLs of your application(s) to which Auth0 can redirect the user after authentication.
-* **Allowed Logout URLs**: the URLs that Auth0 can redirect your users to after logging out;
-* **Allowed Origins (CORS)**: the URLs of the applications running your JavaScript code (prevents same-origin policy errors when using Auth0 from within a web browser)
-* **JWT Expiration (Seconds)**: the amount of time before the Auth0 access tokens expires
-* **Use Auth0 instead of the IdP to do Single Sign On**: if enabled, this setting prevents Auth0 from redirecting authenticated users with valid sessions to the identity provider (such as Facebook, ADFS, and so on)
+The client type should match your application:
```

Comment:
should we extract this as a separate section? (i.e. # Client Types) wdyt? I would also place it before speaking about how to create a client.